eventio/event_io_file.py

```diff
@@ -0,0 +1,65 @@
+import struct
+from .object_header import make_ObjectHeader
+from io import BytesIO
+
+class EventIOObject:
```

Comment:
Nope. `headers`  is a list of headers belonging to this object. An object has multiple headers. 
As I understand it, an eventio file is a kind of tree. As I walk down from header to header, I ultimately arrive at a leave(payload). 

In order to understand the leave, I need all headers, I collected on the way down to the leave. For example, I need  the topmost header to understand the endiannes of the leave, and the downmost header will tell me its length in bytes.